linode/manager
# Fix Uneven Breakpoints in Kubernetes Landing Table

## Description 📝

- FIxes the messed up breakpoints on the Kubernetes Landing Table
- This was likely caused by a combontaion of MUI v5 and React Query for Kubernetes and some conflicts along the way

## Preview 📷

## Before 
![Screen Shot 2023-02-06 at 10 54 45 AM](https://user-images.githubusercontent.com/115251059/217020316-468104cf-15e9-4cad-bb9d-e267271dfc64.jpg)

### After
![Screen Shot 2023-02-06 at 10 54 30 AM](https://user-images.githubusercontent.com/115251059/217020358-08a91793-233c-4eed-bf28-5d19d1babe1e.jpg)

## How to test 🧪

- Go to `http://localhost:3000/kubernetes/clusters`
- Have at least one Kubernetes cluster on your account
- Test resizing the viewport and verify the table breaks as you would expect as you change the viewport
